sdk/storage/azure-storage-datamovement/src/tasks/upload_blob_from_file_task.cpp

```diff
@@ -1,16 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
 #include "azure/storage/datamovement/tasks/upload_blob_from_file_task.hpp"
 
 #include <algorithm>
+#include <limits>
+#include <vector>
 
 #include <azure/core/azure_assert.hpp>
 #include <azure/core/base64.hpp>
 
+#include "azure/storage/datamovement/job_properties.hpp"
 #include "azure/storage/datamovement/scheduler.hpp"
+#include "azure/storage/datamovement/utilities.hpp"
 
 namespace Azure { namespace Storage { namespace Blobs { namespace _detail {
 
   namespace {
-    constexpr uint64_t SingleUploadThreshold = 4 * 1024 * 1024;
+    constexpr uint64_t SingleUploadThreshold = 128 * 1024;
```

Comment:
Why change this? I think 4MB is good.

Reply:
I don't know. We can use 4MB. This value isn't used anyway. Let's revisit when we actually start working on this.